javascripts/src/dataclips.js

```diff
@@ -199,10 +206,15 @@ export default class Dataclips {
                 metadata: { style: xlsx_number_formats.datetime_formatter.id },
               };
             case "duration":
-              return {
-                value: value.as("day"),
-                metadata: { style: xlsx_number_formats.duration_formatter.id },
-              };
+              if (disableSeconds) {
+                return {
+                  value: value.values["hours"] + ":" + formatZeros(Math.abs(value.values["minutes"])),
```

Comment:
what for abs for minutes ? how it's possible minutes have negative value ??

Reply:
Looks like it's a difference calculation between calculated and real tour duration.
I just wrote the line so the behavior is as it is now on master (on master are negative values too).

If we want to change the duration calculations we should open a new task - it's not simple, because we need to discus plenty of cases. I did something similar earlier, that's why I know that.